CONTRIBUTING.md

```diff
@@ -184,7 +184,13 @@ You can also [reach us by email][email].
 
 We below list conventions that we kindly ask all contributors to follow when helping with the content, making fixes or adding/editing lessons. Please consider these hints while you are preparing your pull-requests.
 
-## terminology
+## Terminology
```

Comment:
Shall we follow the ML glossary from google? https://developers.google.com/machine-learning/glossary#training_set